consensus/istanbul/core/core.go

```
@@ -195,7 +188,14 @@ func GetAggregatedSeal(seals MessageSet, round *big.Int) (types.IstanbulAggregat
 	committedSeals := make([][]byte, seals.Size())
 	for i, v := range seals.Values() {
 		committedSeals[i] = make([]byte, types.IstanbulExtraBlsSignature)
-		copy(committedSeals[i][:], v.CommittedSeal[:])
+
+		var commit *istanbul.CommittedSubject
+		err := v.Decode(&commit)
+		if err != nil {
+			panic(fmt.Sprintf("commit: error in decoding committed subject for address %s", hex.EncodeToString(v.Address[:])))
```

Please return an error instead